### PR TITLE
Create Wizard for Creating Member

### LINKED ISSUE #217: Create a wizard for adding individuals from the group
Image

### PR DESCRIPTION
**Pull request by emjay0921**

### CODE REVIEW

**Review by nhatnm0612 — DISMISSED** [review]

**Review by jeremi — CHANGES_REQUESTED** [review]
Can you add the additional details to the wizard ?

**Review by gonzalesedwin1123 — APPROVED** [review]

**Review by jeremi — CHANGES_REQUESTED** [review]
please improve the test coverage before we merge it.

File: spp_registry_wizard/wizard/create_member_wizard.py [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+                role_id = str(role.id)
+                role_id = int(re.search(r"\d+", role_id).group())
+                _logger.info(role_id)
+                if (
+                    role_id
+                    == self.env.ref(
+                        "g2p_registry_membership.group_membership_kind_head"
+                    ).id
+                    and rec.head_member
+                ):
```
Comment (nhatnm0612) [inline spp_registry_wizard/wizard/create_member_wizard.py:212]:
```suggestion
              if (
                  self.env.ref(
                      "g2p_registry_membership.group_membership_kind_head"
                  ) in rec.role
                  and rec.head_member
              ):
```

Comment (nhatnm0612) [inline spp_registry_wizard/wizard/create_member_wizard.py:212]:
```suggestion
                if self.env.ref("g2p_registry_membership.group_membership_kind_head") in rec.role and rec.head_member:
```

File: spp_registry_wizard/wizard/create_member_wizard.py [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+                            if m.isdigit():
+                                kind_str = kind_str + m
+                        if (
+                            int(kind_str)
+                            == self.env.ref(
+                                "g2p_registry_membership.group_membership_kind_head"
+                            ).id
+                        ):
+                            # _logger.info("Head Member: %s " % members.individual.name)
+                            head_member = members.individual.id
```
Comment (nhatnm0612) [inline spp_registry_wizard/wizard/create_member_wizard.py:253]:
```suggestion
                for members in rec.group_id.group_membership_ids:
                    if self.env.ref("g2p_registry_membership.group_membership_kind_head") in members.kind:
                        head_member = members.individual.id
```

**Review by nhatnm0612 — APPROVED** [review]

**Pull request closed without merging**